console/console.js

```diff
@@ -82,13 +82,17 @@ function getInputParamsFromCommandLineIfAny(){
 				printHelp();		
 				process.exit(0);
 			case '-server':
-				if (process.argv[i + 1] == null)
+				const server_arg = process.argv[i + 1];
+				if (server_arg == null)
 				{
 					console.log('Please specify server address to use. Example: node console -server www.remote.com:8888.');
 					console.log('If port is not specified connection will try port 80.');
 					process.exit(0);
 				}
-				server = process.argv[i + 1];
+				else if(server_arg.length > 8 && server_arg.substr(0,8) == 'https://')
```

Comment:
if I run node console -server http://111.111.111.111:4444 fails. I think that should work too.